src/core/dynamic_batch_scheduler.cc

```diff
@@ -113,8 +113,8 @@ DynamicBatchScheduler::Create(
         "Initialization failed for all dynamic-batch scheduler threads");
   }
 
-  sched->completion_promises_ =
-      std::vector<std::shared_ptr<std::promise<void>>>(
+  sched->completion_queue_ =
```

Comment:
completion_queues_ would be clearer since there are multiple queues.

Reply:
Done